Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -193,223 +193,223 @@
+    <string name="submission_dispatcher_tan_code_card_text">Verify your test via manual TAN entry</string>
 
-    <string name="submission_dispatcher_card_tan_tele">TAN telefonisch anfragen</string>
-    <string name="submission_dispatcher_tan_tele_card_text">Bitte rufen Sie uns an, falls Sie Positiv getestet wurden</string>
+    <string name="submission_dispatcher_card_tan_tele">Request TAN by phone</string>
+    <string name="submission_dispatcher_tan_tele_card_text">Please call us if you have tested positive</string>
 
     <!-- Submission Positive Other Warning -->
-    <string name="submission_positive_other_warning_title">Andere warnen</string>
-    <string name="submission_positive_other_warning_headline">So geht es weiter:</string>
+    <string name="submission_positive_other_warning_title">Warn others</string>
+    <string name="submission_positive_other_warning_headline">How it goes on:</string>
     <string name="submission_positive_other_warning_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. \n\nLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. \n\nDuis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</string>
     <string name="submission_positive_other_warning_data_security_body">+Data-Security Text</string>
-    <string name="submission_positive_other_warning_button">Weiter</string>
-    <string name="submission_positive_dialog_confirmation_title">Zufalls-IDs mit der App "Corona-Warn" teilen?</string>
+    <string name="submission_positive_other_warning_button">Next</string>
+    <string name="submission_positive_dialog_confirmation_title">Share random IDs with the "Corona-Warn" app?</string>
     <string name="submission_positive_dialog_confirmation_body">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</string>
-    <string name="submission_positive_dialog_confirmation_positive">Erlauben</string>
-    <string name="submission_positive_dialog_confirmation_negative">Nicht erlauben</string>
+    <string name="submission_positive_dialog_confirmation_positive">Allow</string>
+    <string name="submission_positive_dialog_confirmation_negative">Not allow</string>
```

Comment:
Can this be just "Cancel" or "Don't allow"?

Reply:
Fixed. Changed it to "Don't allow"